support-frontend/assets/pages/contributions-landing/components/ContributionAmount.jsx

```diff
@@ -31,7 +32,7 @@ type PropTypes = {|
   selectedAmounts: SelectedAmounts,
   selectAmount: (number | 'other', CountryGroupId, ContributionType) => (() => void),
   otherAmounts: OtherAmounts,
-  checkOtherAmount: (string, CountryGroupId, ContributionType) => boolean,
+  // eslint-disable-next-line react/no-unused-prop-types
```

Comment:
do we need this?

Reply:
it complains that I'm not using it, even though I am. It's very odd

Reply:
spoke about this in a chat - but just for posterity:

this is a known "false-positive" with this rule: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-unused-prop-types.md#false-positives-sfc

seems like the suggested fix is to assign it to a local variable inside the outer component, and then reference that inside the helper function

Reply:
or keep the eslint ignore ofc! but in that case it's probably worth adding a comment explaining it's a false positive

Reply:
happily, the refactor has made this issue go away because it's no longer within a helper function!